hdrs/extchat.h

```diff
@@ -93,6 +98,7 @@ struct chanuser {
 #define Chanuser_Hide(u) ((CUtype(u) & CU_HIDE) || (IsPlayer(CUdbref(u)) && hidden(CUdbref(u))))
 #define Chanuser_Gag(u) (CUtype(u) & CU_GAG)
 #define Chanuser_Combine(u) (CUtype(u) & CU_COMBINE)
+#define Hidden(d) (d->hide == 1)
```

Comment:
This shouldn't really be in extchat.h (it has nothing to do with channels, and will cause conflicts if this file is included in other files that use Hidden()); better in extchat.c where it's used.